packages/components/tooltip/src/content.vue

```diff
@@ -65,6 +73,9 @@ const ns = useNamespace('tooltip')
 // TODO any is temporary, replace with `InstanceType<typeof ElPopperContent> | null` later
 const contentRef = ref<any>(null)
 const destroyed = ref(false)
+// The el-popper-content should rendered once when el-tooltip is mounted, because
+// sometimes the component's function relies on the props of the slot elements. #17258
+const mounted = ref(false)
```

Comment:
I think the comments here can be adjusted.
`The data of the select and tree-select components comes from the child components rendered by el-popper-content, so when the value of persistent is false, the initial rendering is also performed to correctly echo the default value.`

The variable name can also be modified. `mounted` => `firstRender`

Reply:
Yeah, have updated, Thank you!